win32ss/user/ntuser/menu.c

```diff
@@ -3023,7 +3023,14 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
         if ((y - height) < monitor->rcMonitor.top || y >= monitor->rcMonitor.bottom)
             y = monitor->rcMonitor.bottom - height;
         else
-            y -= height;
+        {
+            AdjHgt = y + UserGetSystemMetrics(SM_CYMENUSIZE) +
```

Comment:
instances of variables should begin not with capital letters 2/4
```suggestion
            adjHgt = y + UserGetSystemMetrics(SM_CYMENUSIZE) +
```

Reply:
Fixed. Thanks.